### PR TITLE
fix: fix ubuntu version get

### LINKED ISSUE #1910: kk create manifest   panic: runtime error: index out of range [1] with length 1
### What is version of KubeKey has the issue?

master

### What is your os environment?

Ubuntu Kinetic Kudu (development branch)

### KubeKey config file

_No response_

### A clear and concise description of what happend.

1. `root@master1:~# ./kk create manifest --ignore-err
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/kubesphere/kubekey/v3/cmd/kk/pkg/artifact.CreateManifest({{0x0, 0x0}, {0xc0005047e0, 0x1a}, {0x0, 0x0}, 0x0, {0x0, 0x0}, 0x0, ...}, ...)
        github.com/kubesphere/kubekey/v3/cmd/kk/pkg/artifact/manifest.go:98 +0x16dc
github.com/kubesphere/kubekey/v3/cmd/kk/cmd/create.(*CreateManifestOptions).Run(...)
        github.com/kubesphere/kubekey/v3/cmd/kk/cmd/create/manifest.go:89
github.com/kubesphere/kubekey/v3/cmd/kk/cmd/create.NewCmdCreateManifest.func1(0xc000795b80?, {0xc00007d170?, 0x1?, 0x1?})
        github.com/kubesphere/kubekey/v3/cmd/kk/cmd/create/manifest.go:57 +0xb7
github.com/spf13/cobra.(*Command).execute(0xc000795b80, {0xc00007d160, 0x1, 0x1})
        github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0xc0007c7b80)
        github.com/spf13/cobra@v1.5.0/command.go:990 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
        github.com/kubesphere/kubekey/v3/cmd/kk/main.go:34 +0xde
`

2. nodeInfo
`nodeInfo:
    architecture: amd64
    bootID: bb78ed61-a2ac-4901-8b51-d72dd44e1b15
    containerRuntimeVersion: containerd://1.6.4
    kernelVersion: 5.19.0-15-generic
    kubeProxyVersion: v1.21.5
    kubeletVersion: v1.21.5
    machineID: aa36372d417751bca63b79160b5b355a
    operatingSystem: linux
    osImage: Ubuntu Kinetic Kudu (development branch)
    systemUUID: aa36372d-4177-51bc-a63b-79160b5b355a
`
3. reason:
cmd/kk/pkg/artifact/manifest.go
`
	osImageArr := strings.Split(node.Status.NodeInfo.OSImage, " ")
		switch strings.ToLower(osImageArr[0]) {
		case "ubuntu":
			id = "ubuntu"
			v := strings.Split(osImageArr[1], ".")
			version = fmt.Sprintf("%s.%s", v[0], v[1])
`

len(v) is 1 and is Kinetic 

### Relevant log output

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by wongearl**
### What type of PR is this?

/kind bug

### What this PR does / why we need it:
fix: fix ubuntu version get
### Which issue(s) this PR fixes:

Fixes #1910

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kk/pkg/artifact/manifest.go [hunk 1]
```diff
@@ -95,7 +95,9 @@ func CreateManifest(arg common.Argument, name string) error {
 		case "ubuntu":
 			id = "ubuntu"
 			v := strings.Split(osImageArr[1], ".")
-			version = fmt.Sprintf("%s.%s", v[0], v[1])
+			if len(v) == 2 {
```
Comment (LinuxSuRen) [inline cmd/kk/pkg/artifact/manifest.go:98]: I prefer using `len(v) >= 2`.

Comment (wongearl) [reply cmd/kk/pkg/artifact/manifest.go:98]: reasonable

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Comment by littleBlackHouse** [comment]
@all-contributors please add @wongearl   for code

**Pull request merged**